tensorflow/python/ops/array_ops.py

```diff
@@ -1252,6 +1252,52 @@ def sparse_mask(a, mask_indices, name=None):
     return ops.IndexedSlices(out_values, out_indices, a.dense_shape)
 
 
+def unique(x, out_idx=None, name=None, axis=None):
```

Comment:
Can we do without this wrapper?

Reply:
@drpngx It should be possible if we add a line `    .Input("axis: int64")` to array_ops.cc:
```
REGISTER_OP("Unique")
    .Input("x: T")
    .Output("y: T")
    .Output("idx: out_idx")
    .Attr("T: type")
    .Attr("out_idx: {int32, int64} = DT_INT32")
    .SetShapeFn([](InferenceContext* c) {
      c->set_output(0, c->Vector(InferenceContext::kUnknownDim));
      c->set_output(1, c->input(0));
      return Status::OK();
    })
```

Though that means one additional input for the existing API. Will that be OK?

Reply:
Right, that's the old op. So we need the wrapper just to rename into v2, but why the rest of the code? I don't think it's needed. Then we can just have `unique = gen_array_ops.unique_v2`.